### ISSUE TITLE
[Bug report] `Ctrl` doesn't work on Windows 10 

### ISSUE THREAD

**Issue by nhan000** [Qwyntex/doubleshift#10]
As in the title. I couldn't make it work for `Ctrl`. 
`Alt`, `Shift`, `F1` work fine.

![image](https://user-images.githubusercontent.com/85216095/183310713-c3479b08-23c3-4ff5-9882-4f7a7dc8917b.png)

I think it's much better if the plugin can recognize key press when set up the hotkey (like the built-in Hotkeys setting). It's a lot more users friendly. For example, I don't know if the issue in this bug report is actually a bug or I just typed the `Ctrl` key name incorrectly and therefore, the plugin doesn't recognize the keypress.

**Comment by Qwyntex** [comment]
Ctrl is like the Windows key a modifier key can not be used with my current keylistener. I will eventually rewrite this part but for now this does not work.

**Comment by nhan000** [comment]
Thanks a lot! Do you mind adding a line in the Readme document specifying that it supports many keys other than `Shift` (I missed this due to the name of the plugin), and doesn't work with `Ctrl` and `Windows` keys?

**Comment by Qwyntex** [comment]
I see no problem with that

**Comment by ghost** [comment]
Greetings! I am happy to report that, on my installation, at least, the Ctrl key _does_ work perfectly well—at least on Windows 10. _However_, in order to get it to work, ==you need to set the name of the key to `Control` instead of `Ctrl`==; that is, use the unabbreviated name for the key.

I _was_ of the opinion that it was impossible to assign `Ctrl` as a hotkey when I initially set up the _Quick Switcher++_ community plugin to open upon double-pressing it, but after changing the key name as described above, it worked as expected. Whether this will work for **everyone** on Windows 10 or not I don't know, but it should theoretically.

Hope that solves your issue. Have a good day, everyone! :smile:

---
TL;DR:

If you're curious, as I am, as to the reason for this bug, we can be reasonably sure the issue originates either **(1)** in Obsidian's internal logic, having to do with the strings associated with each key-code, or **(2)** with a certain JavaScript API (maybe Electron?) that Obsidian uses. Of course, I could be mistaken; I have little experience with JavaScript or the Electron API. But if they work similarly to other languages (e.g., C/++, Asm, Java), which I _am_ familiar with, then given that the Control key must be explicitly typed, it is probably so.

Because of this, it may be worthwhile to try alternate values for the key name string(s) of other modifier keys, too. Perhaps, for example, the `Windows` key is internally assigned to a string such as `Meta` or `Left-Meta`. Simple experimentation can often bear dramatically helpful results.

Finally, Please bear in mind that these _"key name bugs"_ are no fault of Qwyntex (the _Doubleshift_ plugin's author),  whose code is stable and well-written. Qwyntex is really great at following up on issues and fixing things ASAP, an attribute I **highly** admire, as many developers (myself included) are not _always_ so efficient.

**Comment by Qwyntex** [comment]
I have just tested it on my Linux system and it seems to work there as well so i assume it will work on mac too. I honestly feel kind of stupid for not trying `Control` instead of `Ctrl`, so thank you. As for the meta key, I don't think that this works but fell free to try it out. The issue regarding a disabled plugin crashing the settings page is fixed as of the latest release.

**Comment by ghost** [comment]
Thank you for your kind comment, Qwyntex, and don't worry about not trying alternatives to `Ctrl`. Actually it took me a frustratingly long day wracking my brain to even _think_ about trying a different string like `Control`. Sometimes things just take a different perspective, or a different "set of eyes" hehe.

And thank you for your quick responses and professional demeanor/actions. I like the new releases and I've been using some of your other _amazingly_ brilliant plugins as well. They are truly helpful and have assisted in cutting down the time I spend setting certain things up or searching for something. It's much appreciated!

Your most gratefully,
    -frostbyte
